bravado/http_future.py

```diff
@@ -13,7 +13,15 @@
 
 from bravado.config_defaults import REQUEST_OPTIONS_DEFAULTS
 from bravado.exception import BravadoTimeoutError
+from bravado.exception import HTTPServerError
 from bravado.exception import make_http_exception
+from bravado.response import BravadoResponse
+
+
+FALLBACK_EXCEPTIONS = (
```

Comment:
Should we automatically catch exceptions which are mapped to 5xx status codes? Thinking about `bravado.exception.status_map`, here:
https://github.com/Yelp/bravado/blob/cf8ccc26d2f151d67da0f2eab6e0ac0eb9a20eec/bravado/exception.py#L12